### DISCUSSION TITLE
Add setting to disable Login/Sign Up buttons

### DISCUSSION THREAD

**Discussion by astrolemonade** [Kong/insomnia#6396] [Ideas]
Add setting to disable Login/Sign Up buttons
![image](https://github.com/Kong/insomnia/assets/35623921/1405b8d8-b9a9-4359-a7fd-47100a87d156)
There are multiple times I misclicked them and I think I am not the only one.

**Comment by kobenguyent** [comment]
perhaps this https://github.com/Kong/insomnia/pull/6533
